### ISSUE TITLE
Prompting for password for p12 when there is none

### ISSUE THREAD

**Issue by jlubeck** [noodlewerk/NWPusher#41]
I created my certificates using PEM which creates p12 files without passwords. If I use the p12 on others services it works fine, but Pusher seems to ask for a password, and if I leave it blank i get this error "Unable to read p12 file: PKCS12 data wrong password" and it doesn't show up in the list of certificates.

**Comment by leonardvandriel** [comment]
Hi @jlubeck,

Thanks for reporting this issue. I was able to reproduce it, but unable to fix.

Unfortunately Apple's p12 import function (https://developer.apple.com/reference/security/1396915-secpkcs12import) does not seem to support files without password. I did add support for it in the Mac app though (commit ff57680), but it will results in a `password required` error.

There's an Open Radar post on this issue: http://www.openradar.me/22909471. This is referenced on stackoverflow: http://stackoverflow.com/questions/39392338/secpkcs12import-when-p12-requries-an-empty-passphrase

For now I suggest adding a password.
